### ISSUE TITLE
Generating color palette... Error: Failed to generate color palette

### ISSUE THREAD

**Issue by imTHAI** [Gords/GIFtube#2]
Hello,

I just tried and with the first video I try, I get
```
Generating color palette...
Error: Failed to generate color palette
```

I'm under MacOS 15.0 
Any idea? If I can give you any logs for the debug, please tell me what you'd need.
Thanks

**Comment by Gords** [comment]
Hello! 

I tried it on macOS 14 just a couple of weeks ago, hmmm, if you could please share the logs that would be very helpful!

**Comment by imTHAI** [comment]
Hello,

After others tests I found the problem I think.
When it asked for: ":Enter the start time (in seconds):" I didn't enter anything and pressed enter, assuming it would start at the beginning by default. 

May I suggest the script exits right there, and tells that a value is mandatory ?
Or maybe better, to set a default value of zero (or whatever) ?  
```
read -p "Enter the start time (in seconds. Default is 0): " stt
stt=${stt:-0}
```
(and maybe for other mandatories values if any).

Just a suggestion. It was my fault anyway, I tested too quickly.
